### ISSUE TITLE
".nzb Backup Folder" config is cleared if it cannot be created/accessed at startup

### ISSUE THREAD

**Issue by Cpuroast** [sabnzbd/sabnzbd#1834]
In Sab 3.2.0, not sure about older versions, the ".nzb Backup Folder" config is cleared if it cannot be created/accessed at startup.

**Comment by Safihre** [comment]
I don't know if that's a bug or a feature 😃

**Comment by thezoggy** [comment]
Prob fallout from jcfp fixes on #

**Comment by jcfp** [comment]
> its by design after #1765

I fail to see how requiring valid scripts would affect the NZB backup setting?

Given that these backups are disabled by default and not needed for anything else, I suspect this behaviour is a design choice.

**Comment by Safihre** [comment]
This setting is of type `OptionDir`, which has `create=True` as default. Because of that it will check if the directory exists and is writeable.
It has always been like that.
Indeed, this is a design choice to prevent crashes at a later stage due to unavailable paths.
